rental.rb

```diff
@@ -0,0 +1,20 @@
+require 'time'
+
+class Rental
+  attr_reader :person, :book
+  attr_accessor :date
+
+  def initialize(person, book)
+    @date = DateTime.now.strftime('%Y/%m/%d %H:%M')
+    @person = person
+    @book = book
+  end
+
+  def assign_book
+    @book.rentals << self unless @books.rentals.include? self
+  end
+
+  def assign_person
+    @person.rentals << self unless @person.rentals.include? self
+  end
```

Comment:
- [ ] Instead of doing this here, please modify the constructor of your `Rental` class so that the `belongs-to` relationship between `Book/Rental` and `Person/Rental` can be implemented. Please see **HINT** below 👇 
```ruby
def initialize(person, book)
  person.rentals << self
  .
  .
  .
end
```